test-integration/util/user.ts

```diff
@@ -32,13 +32,15 @@ export async function createRandomID() {
 
     await user.createPerson();
     await user.updateKeys(password);
+    const chainID = randomString(8);
```

Comment:
this is not right, chainId comes from the getInfo endpoint on the blockchain API